lib/styles/settings/typography/_tokens.scss

```diff
@@ -1,7 +1,7 @@
 @import 'variables';
 
 @mixin DisplayHeadingMRegular() {
-	--token: displayHeading-m-regular;
+	--token: displayheading-m-regular;
```

Comment:
we've lost camel case now

Reply:
I'm feeling that we have the same bug now. Maybe we should add cases we want to cover to unit-tests and iterate until tests pass?

Reply:
We lost but lint is not accepting camel case in css values, it fix it automatically. Yesterday's CI didn't passed by this case. I don't want to modify our lint rules, because they are global.

Reply:
maybe it's possible to ignore this rule for one file only?

Reply:
Nothing works, tips from here https://stackoverflow.com/questions/34764287/turning-off-eslint-rule-for-a-specific-file or rule in `stylelint.config.js`
```
		'value-keyword-case': [
			'lower',
			{
				ignoreFunctions: ['/^displayHeading*$/', 'displayHeading-m-regular'],
			},
		],
```

Still I don't see the point of disabling it, it's just visual information seen in devtools.

Reply:
according to docs you should use `ignoreKeywords`: https://stylelint.io/user-guide/rules/value-keyword-case/#ignorekeywords-regex-regex-non-regex or even better if you use `ignoreProperties`: https://stylelint.io/user-guide/rules/value-keyword-case/#ignoreproperties-regex-regex-non-regex

Reply:
This works:
```
'value-keyword-case': [
			'lower',
			{
				ignoreKeywords: ['/^displayHeading*/'],
			},
		],
```

@kvas-damian I leave it up to you to decide whether to use it and undo the changes.